static/app/gettingStartedDocs/javascript/react.tsx

```diff
@@ -187,13 +187,13 @@ const onboarding: OnboardingConfig = {
       id: 'react-router',
       name: t('React Router'),
       description: t(
-        'Configure routing, so Sentry can generate parameterized transaction names for a better overview in Performance Monitoring.'
+        'Configure routing, so Sentry can generate parameterized transaction names for a better overview in Tracing.'
```

Comment:
```suggestion
        'Configure routing, so Sentry can generate parameterized transaction names for a better overview on the Performance page.'
```
parameterized transaction names still show up on the Performance page. So we should leave this as is.